drivers/input/input_ft5336.c

```diff
@@ -18,7 +18,7 @@
 LOG_MODULE_REGISTER(ft5336, CONFIG_INPUT_LOG_LEVEL);
 
 /* FT5336 used registers */
-#define REG_TD_STATUS		0x02U
+#define REG_TD_STATUS	0x02U
```

Comment:
check your identations, this and EVENT_NONE are being thrown off for some reason, the new one have spaces instead of tab, just keep it coherent with the rest of the file

Reply:
My fault. I had tabs at 4 instead of 8.